servlet25/src/main/java/brave/webmvc/TracingFactoryBean.java

```diff
@@ -0,0 +1,38 @@
+package brave.webmvc;
+
+import brave.Tracing;
+import brave.context.slf4j.MDCCurrentTraceContext;
+import org.springframework.beans.factory.FactoryBean;
+import zipkin.Span;
+import zipkin.reporter.Reporter;
+
+/** Spring XML config does not support chained builders. This exposes what we use. */
+public class TracingFactoryBean implements FactoryBean<Tracing> {
```

Comment:
thinking of making `brave-spring-beans` which has this fully fleshed out, so legacy apps can add jars and config with only XML